resources/function_help/json/round

```diff
@@ -2,15 +2,15 @@
   "name": "round",
   "type": "function",
   "groups": ["Math"],
-  "description": "Rounds a number to number of decimal places.",
+  "description": "Rounds a number to number of decimal places. The rounding is done to the nearest number. If you want to specifically round up or down use the ceil and floor functions.",
```

Comment:
I think we should use passive voice here, to keep consistency with other function help:

```suggestion
  "description": "Rounds a number to number of decimal places. The rounding is done to the nearest number. The ceil or floor functions can be used if rounding up or down is required.",
```

Reply:
@nyalldawson Better indeed. Done. Squashed commits and force pushed. Thank you!